### ISSUE TITLE
complele project report on descrete structures

### ISSUE THREAD

**Issue by beinganukul** [subash9860/Discrete-Structures#1]
are you planning to push all of the codes???

**Comment by subash9860** [comment]
yes, i am working on this. i want you to help me for doing this assignment.

**Comment by subash9860** [comment]
> are you planning to push all of the codes???

yes, i am working on this. i want you to help me for doing this assignment.
